### PR TITLE
gh-84461: Document how to install SDKs manually (GH-93844)

### PR DESCRIPTION
**Pull request by tiran**

### CODE REVIEW

**Comment by carlbordum** [comment]
I have a feeling these instructions will get outdated quite quickly. Perhaps its better to link to the official install instructions for Emscripten, WASI-SDK, wasmtime and WASIX? Otherwise, LGTM

**Comment by tiran** [comment]
Every sections links to the official documentation for each project, too. The official docs do not cover some aspects, e.g. how to install a specific version of wasmtime. Except for version numbers I do not expect any instructions to get outdated any time soon.

File: Tools/wasm/README.md [hunk 1]
```diff
@@ -338,3 +338,81 @@ Feature detection flags:
...
+curl -sSf -L -o ~/install-wasmtime.sh https://wasmtime.dev/install.sh
+chmod +x ~/install-wasmtime.sh
+~/install-wasmtime.sh --version v0.36.0
+ln -srf -t /usr/local/bin/ ~/.wasmtime/bin/wasmtime
+```
+
+### Install [WASIX](https://github.com/singlestore-labs/wasix)
+
+```shell
+git clone -b cpython https://github.com/tiran/wasix.git ~/wasix
```
Comment (brettcannon) [inline Tools/wasm/README.md]: Why the clone from your fork? Or was that a copy-and-paste mistake?

```suggestion
git clone https://github.com/singlestore-labs/wasix.git ~/wasix
```

**Review by brettcannon — APPROVED** [review]
Other than a question about using a personal fork of WASIX, it LGTM!

File: Tools/wasm/README.md (hunk 1 above)
Comment (tiran) [reply Tools/wasm/README.md]: The upstream repo was missing my Makefile improvements. They merged my PR a couple of hours ago.

**Review by pmp-p — APPROVED** [review]

**Pull request merged**